### PR TITLE
Release 1.5.5

### PR DESCRIPTION
**Pull request by CyberDem0n**
* bump version
* update release notes

### CODE REVIEW

File: docs/releases.rst [hunk 1]
```diff
@@ -3,6 +3,44 @@
 Release notes
 =============
 
+Version 1.5.5
+-------------
+
+This version intoduces possibility of automatic reinit of the former master, improves patronictl list output and fixes a number of bags.
```
Comment (redbaron) [inline docs/releases.rst]: > b**a**gs

File: docs/releases.rst [hunk 2]
```diff
@@ -3,6 +3,44 @@
...
+
+**New features**
+
+- Add support of `PATRONI_ETCD_PROTOCOL`, `PATRONI_ETCD_USERNAME` and `PATRONI_ETCD_PASSWORD` environment variables (Étienne M)
+
+  Before it was possible to configure them only in the config file or as a part of `PATRONI_ETCD_URL`, which is not always convinient.
```
Comment (redbaron) [inline docs/releases.rst]: > conv**i**nient

File: docs/releases.rst [hunk 3]
```diff
@@ -3,6 +3,44 @@
...
+
+  It helps to detect stale replicas. In addition to that, `Host` will include ':{port}' if the port value isn't default or there is more than one member running on the same host.
+
+- Create a headless service associated with the $SCOPE-config endpoint (Alexander)
+
+  The "config" endpoint keeps information about the cluster-wide Patroni and Postgres configuration, history file, and last but the most important, it holds the `initialize` key. When the Kubernetes master node is restarted or upgraded, it removes endpoints without services. The headless service will prevent it from being removed.
+
+**Bug fixes**
+
+- Adjust the read timeout for the leader watch blocking query (Alexander)
```
Comment (avaczi) [inline docs/releases.rst]: 'watch blocking query'?  The description doesn't hint anything similar

Comment (CyberDem0n) [reply docs/releases.rst]: This is the Consul [terminology](https://www.consul.io/api/index.html#blocking-queries)...

**Pull request merged**